docs/rfcs/0623-redis-backend.md

```diff
@@ -0,0 +1,237 @@
+- Proposal Name: "redis_backend"
```

Comment:
I prefer renaming it to `redis_service`.

We support the redis service, and the `backend` is used to implement the support.

Reply:
By the way, we expect to use

```markdown
`redis_service`
```